api/conf/schema.json

```diff
@@ -3240,6 +3285,12 @@
 				"type": "object"
 			}
 		},
+		"server-info": {
```

Comment:
How about the `attr_schema`?

https://github.com/apache/apisix/blob/9a07ee8f825dcc428e9255096629ad74e7968d1c/apisix/plugins/server-info.lua#L35-L53

```
local attr_schema = {
    type = "object",
    properties = {
        report_interval = {
            type = "integer",
            description = "server info reporting interval (unit: second)",
            default = default_report_interval,
            minimum = 60,
            maximum = 3600,
        },
        report_ttl = {
            type = "integer",
            description = "live time for server info in etcd",
            default = default_report_ttl,
            minimum = 3600,
            maximum = 86400,
        }
    }
}
```